lib/experimental/Information/Communities/Post/PostEvent/index.tsx

```diff
@@ -0,0 +1,71 @@
+import { DateAvatar } from "@/experimental/Information/Avatars/DateAvatar"
+import { withSkeleton } from "@/lib/skeleton"
+import { Skeleton } from "@/ui/skeleton"
+
+type PostEventProps = {
+  title: string
+  image?: string
+  place: string
+  date: {
+    hour: string
+    day: number
+    month: string
+  }
```

Comment:
I think we should agree on a format to pass these dates to our components in a simpler way, maybe like "YYYY-MM-DD" which is how we have it in backend, and then we just parse it inside and extract whatever we want

Reply:
Good point, I have no idea how we retrieve this info, so this is gold for me, haha. Maybe this is a topic to discuss on Slack. Do you mind if I open a thread there?

Reply:
Let's also consider getting a `Date` object instead and get all the dates from it